proxy/files/etc/nginx/conf.d/nginx.conf.template

```diff
@@ -266,7 +266,9 @@ server {
     proxy_pass  $internalUrl/api/csp/csp-report;
   }
 
-  <% if ENV.key?("KEYCLOAK_URL") && ENV["KEYCLOAK_URL"] != "" %>
+  <% if ENV.key?("KEYCLOAK_CONFIGURATION") && ENV["KEYCLOAK_CONFIGURATION"] != "" %>
+  <%= ENV["KEYCLOAK_CONFIGURATION"] %>
+  <% elsif ENV.key?("KEYCLOAK_URL") && ENV["KEYCLOAK_URL"] != "" %>
```

Comment:
I'd maybe add a comment here to describe the rationale behind allowing a complete override vs. the pre-defined config for posterity.

Reply:
The pre-defined was left there for backwards combability so it supports both configurations without failing.

Reply:
OK, so was the idea to remove it at some later point completely (that's OK, just affects whether I'd like the comment or not)?

Reply:
We could leave it as example there, but if we go this way, we will not be using it anymore when this is installed on all systems and becomes unnessary and can be removed then. This way we do not have to worry synching the infra with this commit.

I can leave deprecation comment there.